src/libs/getOperatingSystem/index.native.js

```diff
@@ -0,0 +1,17 @@
+import {Platform} from 'react-native';
+import CONST from '../../CONST';
+
+/**
+ * Reads the current operating system for native platforms.
+ * @return {String | null}
+ */
+export default () => {
+    switch (Platform.OS) {
```

Comment:
We usually split stuff like this into an `index.android.js` and `index.ios.js` files instead of use `Platform` + `index.native.js`.